internal/databases/mysql/binlog_find_handler.go

```diff
@@ -0,0 +1,23 @@
+package mysql
+
+import (
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+	"github.com/wal-g/wal-g/utility"
+
+	"github.com/wal-g/tracelog"
+)
+
+func HandleBinlogFind(folder storage.Folder, gtid string) {
+	db, err := getMySQLConnection()
+	tracelog.ErrorLogger.FatalOnError(err)
+	defer utility.LoggedClose(db, "")
+	flavor, err := getMySQLFlavor(db)
+	tracelog.ErrorLogger.FatalOnError(err)
+	if gtid == "" {
+		gtid, err = getMySQLGTIDExecuted(db, flavor)
```

Comment:
Is there at least single valid usage for this fallback?
I think that users may use it in a way that will break their backup-recovery process.